test/e2e_node/pod_conditions_test.go

```diff
@@ -125,7 +126,7 @@ func runPodFailingConditionsTest(f *framework.Framework, hasInitContainers, chec
 			// Verify PodInitialized is set if init containers are not present (since without init containers, it gets set very early)
 			initializedTime, err := getTransitionTimeForPodConditionWithStatus(p, v1.PodInitialized, true)
 			framework.ExpectNoError(err)
-			framework.ExpectNotEqual(initializedTime.Before(scheduledTime), true, fmt.Sprintf("pod without init containers is initialized at: %v which is before pod scheduled at: %v", initializedTime, scheduledTime))
+			gomega.Expect(initializedTime.Before(scheduledTime)).ToNot(gomega.BeTrue(), fmt.Sprintf("pod without init containers is initialized at: %v which is before pod scheduled at: %v", initializedTime, scheduledTime))
```

Comment:
Can you change this to not use BeTrue? See https://github.com/kubernetes/kubernetes/issues/105678.

FYI: https://github.com/kubernetes/kubernetes/pull/119558/files#diff-ca60431bd6cf2bd182b588d9b8c9135be21d1a0d49c7c2a8b60826620b4c83cfR151

Reply:
Resolved